Gemfile.lock

```diff
@@ -224,4 +224,4 @@ DEPENDENCIES
   yard
 
 BUNDLED WITH
-   1.17.3
+   2.0.1
```

Comment:
This probably should be a separate PR. If you do go to 2.0, you'll have to install it on the machine before deploying

Reply:
ja, aware.  been doing that for some things already today.  If you want me to split the PR, i will, but i'll deploy 'em both together anyway.

Reply:
Okay, whatever you want. I'll approve.